packages/env/lib/init-config.js

```diff
@@ -176,3 +237,29 @@ RUN echo 'xdebug.mode=${ enableXdebug }' >> /usr/local/etc/php/php.ini
 RUN echo '${ clientDetectSettings }' >> /usr/local/etc/php/php.ini
 	`;
 }
+
+/**
+ * Generates the Dockerfile used by wp-env's `cli` and `tests-cli` instances.
+ *
+ * @param {string} image The base docker image to use.
+ *
+ * @return {string} The dockerfile contents.
+ */
+function cliDockerFileContents( image ) {
+	return `FROM ${ image }
+
+# Switch to root so we can create users.
+USER root
+	
+# Create the host's user so that we can match ownership in the container.
+ARG HOST_USERNAME
+ARG HOST_UID
+ARG HOST_GID
+# When the IDs are already in use we can still safely move on.
+RUN addgroup -g $HOST_GID $HOST_USERNAME || true
+RUN adduser -h /home/$HOST_USERNAME -G $( getent group $HOST_GID | cut -d: -f1 ) -u $HOST_UID $HOST_USERNAME || true
```

Comment:
why are these different from the ones in the wordpress file above? (E.g. useradd vs adduser, and the other changes for setting the user)

Reply:
For some reason, the WordPress and CLI containers use different operating systems. The `wordpress` container uses Ubuntu while the `wordpress:cli` container uses Alpine. This requires us to have different handling to accomplish the same thing in both operating systems.

Reply:
That's unfortunate 🙃